### PR TITLE
adding tutorial

### PR DESCRIPTION
**Pull request by mhajij**
adding DeepCell tutorial and fixing some docstring, making it more clear

### CODE REVIEW

**Review by ninamiolane — CHANGES_REQUESTED** [review]
Some typos: e.g. parmater
DeepCell.ipynb --> deepcell.ipynb to respect python guidelines in naming files.

File: topoembedx/classes/higher_order_laplacian_eigenmaps.py [hunk 1]
```diff
@@ -43,13 +43,35 @@ def fit(
...
-        neighborhood_type : str, optional
-            The type of neighborhood to use, by default "adj".
-        neighborhood_dim : dict, optional
-            The dimension of the neighborhood to use.
-            Default: {"adj": 0, "coadj": -1}
+        complex : TopoNetX object
+            A complex object. The complex object can be one of the following:
+            - CellComplex
+            - CombinatorialComplex
+            - CombinatorialComplex
```
Comment (ninamiolane) [inline topoembedx/classes/higher_order_laplacian_eigenmaps.py:50]: CombinatorialComplex appears twice

**Comment by mhajij** [comment]
@ninamiolane lint is failing in the tutorial should we disable that there or what actions do you recommend ?

**Comment by ninamiolane** [comment]
@mhajij run `black .` in your command line.

**Review by ninamiolane — APPROVED** [review]
Wow, the tutorial is awesome!! Thanks!

**Comment by gurug-dev** [comment]
this PR also fixes #23.

**Pull request merged**